apps/web/app/future/workflows/[workflow]/page.tsx

```diff
@@ -0,0 +1,43 @@
+import { _generateMetadata } from "app/_utils";
+import { WithLayout } from "app/layoutHOC";
+import { type GetServerSidePropsContext } from "next";
+import { headers, cookies } from "next/headers";
+import { notFound } from "next/navigation";
+import { z } from "zod";
+
+import LegacyPage from "@calcom/features/ee/workflows/pages/workflow";
+
+import { buildLegacyCtx } from "@lib/buildLegacyCtx";
+
+const querySchema = z.object({
+  workflow: z.string(),
+});
+
+export const generateMetadata = async ({ params }: { params: Record<string, string | string[]> }) => {
+  const { workflow } = await getProps(
+    buildLegacyCtx(headers(), cookies(), params) as unknown as GetServerSidePropsContext
+  );
+  return await _generateMetadata(
+    () => workflow ?? "Untitled",
+    () => ""
+  );
+};
+
+async function getProps(context: GetServerSidePropsContext) {
+  const safeParams = querySchema.safeParse(context.params);
+
+  console.log("Built workflow page:", safeParams);
```

Comment:
Do we need this log?

Reply:
We copied it from the legacy page as it is. Please let us know if it can be deleted.